.prettierrc

```diff
@@ -1,7 +1,26 @@
 {
-	"useTabs": true,
 	"tabWidth": 4,
+	"useTabs": true,
+	"printWidth": 80,
 	"semi": true,
 	"singleQuote": false,
-	"printWidth": 80
+	"plugins": ["@ianvs/prettier-plugin-sort-imports"],
+	"importOrder": [
+		"^(react/(.*)$)|^(react$)",
+		"^(next/(.*)$)|^(next$)",
+		"<THIRD_PARTY_MODULES>",
+		"",
+		"^types$",
+		"^~/types/(.*)$",
+		"^~/config/(.*)$",
+		"^~/lib/(.*)$",
+		"^~/hooks/(.*)$",
+		"^~/components/ui/(.*)$",
+		"^~/components/(.*)$",
+		"^~/styles/(.*)$",
+		"^~/app/(.*)$",
```

Comment:
Issue: I don't think these particular imports fit the folder structure of our site.